types/git-username/index.d.ts

```diff
@@ -5,8 +5,10 @@
 
 export = gitUsername;
 
+declare function gitUsername(options?: gitUsername.BaseOptions): string | null;
 declare function gitUsername(options: gitUsername.StrictOptions): string;
-declare function gitUsername(options?: string | gitUsername.BaseOptions): string | null;
+declare function gitUsername(cwd: string, options?: gitUsername.BaseOptions): string | null;
+declare function gitUsername(cwd: string, options?: gitUsername.StrictOptions): string;
```

Comment:
The correct typings for this should never return `null` when `strict` is set to `true`. To achieve this, the typings should look something like the following:

```suggestion
declare function gitUsername(options?: gitUsername.BaseOptions): string | null;
declare function gitUsername(cwd: string): string | null;
declare function gitUsername(cwd: string, options: gitUsername.StrictOptions): string;
// tslint:disable-next-line unified-signatures
declare function gitUsername(cwd: string, options: gitUsername.BaseOptions): string | null;
```